internal/printer/print-to-js.go

```diff
@@ -439,14 +439,20 @@ func render1(p *printer, n *Node, opts RenderOptions) {
 			if transform.HasAttr(n, transform.TRANSITION_ANIMATE) {
 				animationName = transform.GetAttr(n, transform.TRANSITION_ANIMATE).Val
 			}
-			transitionName := ""
+			transitionExpr := ""
 			if transform.HasAttr(n, transform.TRANSITION_NAME) {
-				transitionName = transform.GetAttr(n, transform.TRANSITION_NAME).Val
+				attr := transform.GetAttr(n, transform.TRANSITION_NAME)
+				switch attr.Type {
+				case astro.QuotedAttribute:
+					transitionExpr = fmt.Sprintf(`"%s"`, attr.Val)
+				case astro.ExpressionAttribute:
+					transitionExpr = fmt.Sprintf(`(%s)`, attr.Val)
```

Comment:
Probably want to handle `TemplateLiteralAttribute` as well?

Reply:
Ah ok, will do that. didn't even realize we supported that.

Reply:
Added.